CHANGELOG.md

```diff
@@ -19,6 +19,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Multiple solutions are allowed for a single symbolic expression
 - Aliasing works much better for symbolic and concrete addresses
 - Constant propagation for symbolic values
+- Allow reading deployedBytecode.object from the forge JSON as --code or --code-a/-code-b
```

Comment:
Do you mean `--code-file` and `--code-file-a/--code-file-b` here?

Reply:
Oops yes, fixed!